### PR TITLE
fix dependencies: remove the next tag

### PR DESCRIPTION
**Pull request by pelord**
Remove the next mention to the package.json for the v5.0.0

### CODE REVIEW

**Comment by pelord** [comment]
This PR has not been tested.

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "eslint-plugin-react": "^7.24.0",
     "eslint-plugin-react-hooks": "^4.2.0",
     "extensionless": "^1.9.9",
-    "geostyler-style": "9.0.0-next.1",
+    "geostyler-style": "9.1.0",
```
Comment (KaiVolland) [inline package.json:60]:
```suggestion
    "geostyler-style": "9.2.0",
```

**Review by KaiVolland — APPROVED** [review]
Looks good to me. I'd suggest to update to 9.2.0 immediately.

**Comment by KaiVolland** [comment]
Seems like there are some issues with the lock file here. If ok for you I'll do it.

**Comment by KaiVolland** [comment]
Superseeded by #484

**Comment by pelord** [comment]
> Seems like there are some issues with the lock file here. If ok for you I'll do it.

no problem, thanks a lot!

**Pull request closed without merging**